### ISSUE TITLE
Could you integrate `attrib +s` to solve symlink problem?

### ISSUE THREAD

**Issue by flyskywhy** [wishstudio/flinux#79]
I made a `flinux.git` (https://github.com/flyskywhy/flinux) from your `flinux-archlinux.7z`, and added a `green_install_1st_enable_symlink.bat` to sovle the running problem when a new software installed from `pacman -S` if a symlink `.so` file is needed, or can't run `flinux.git` itself when just after git clone it.
Could you integrate this symlink function to your flinux source code? e.g. monitor `pacman -S` and solve the symlink problem in flinux source code.

**Comment by wishstudio** [comment]
I do not fully understand your situation. flinux creates (emulated) symlink files with a system attribute. This should be done automatically and transparently for user. If after `git clone` it does not work, then maybe it's caused by the git removing the system attribute or something? I need more info on this one.

**Comment by flyskywhy** [comment]
Yes, in original `flinux-archlinux.7z`, system attribute was done automatically yet. But when extract flinux-archlinux.7z, and after `pacman -Syu`, then, for example, the upgraded `archlinux/usr/lib/libpcre.so.1` will lose system attribute, the new added `archlinux/usr/lib/libsqlite3.so.0` don't have system attribute, so any software depend on these `lib*.so.*` will not run.

Yes, git can't commit the system attribute into git repository, so after `git clone`, binary `flinux.git` can't run directly, but despite the git version binary flinux, the original version `flinux-archlinux.7z` will not work well after `pacman -Syu` as described above.
